src/Features/Core/Portable/ChangeSignature/Parameter.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.ChangeSignature
+{
+    internal abstract class Parameter
```

Comment:
do we actually need this?  can we not just pass around IPArameters?  Just in the case where we need to make a new one we make a CodeGenerationParameterSymbol?

Reply:
Filed as https://github.com/dotnet/roslyn/issues/42687